content/src/content/jcr_root/apps/core/wcm/components/commons/editor/dialog/childreneditor/v1/childreneditor/clientlibs/js/childreneditor.js

```diff
@@ -252,25 +252,23 @@
                 }
 
                 Coral.commons.ready(that._elements.self, function(item) {
-                    // As a reordering of the multifield also triggers the coral-collection:remove event we have to add
-                    // a check for moved items so the prompt get only shown on a real remove action.
-                    var movedItem;
-
                     that._elements.self.on("coral-multifield:itemorder", function(event) {
-                        movedItem = event.detail.item.dataset["name"];
                     });
```

Comment:
why do you need to handle this event, if nothing is happening here anymore?

Reply:
Kept it just in case. But it is no longer needed.